src/Uno.UI/UI/Xaml/FrameworkElement.Skia.cs

```diff
@@ -0,0 +1,306 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Uno.Disposables;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Extensions;
+using Uno;
+using Uno.Logging;
+using Windows.UI.Xaml.Controls;
+using Windows.Foundation;
+using View = Windows.UI.Xaml.UIElement;
+using System.Collections;
+using Microsoft.Extensions.Logging;
+using Uno.UI.Xaml;
+
+namespace Windows.UI.Xaml
+{
+	public partial class FrameworkElement : IEnumerable
+	{
+		private Size _actualSize;
+
+		private readonly static Thickness _thicknessCache = Thickness.Empty;
+
+		public FrameworkElement()
+		{
+			_log = this.Log();
+			_logDebug = _log.IsEnabled(LogLevel.Debug) ? _log : null;
+			Initialize();
+		}
+
+		public new bool IsLoaded
+		{
+			get => base.IsLoaded; // The IsLoaded state is managed by the UIElement, FrameworkElement only makes it publicly visible
+			set => base.IsLoaded = value;
+		}
+
+		bool IFrameworkElementInternal.HasLayouter => true;
+
+		partial void Initialize();
+
+		protected internal readonly ILogger _log;
+		private protected readonly ILogger _logDebug;
+
+		partial void OnLoadingPartial();
+		private protected virtual void OnPostLoading()
+		{
+		}
+		public bool HasParent()
+		{
+			return Parent != null;
+		}
+
+		//protected internal override void OnInvalidateMeasure()
+		//{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)